test/e2e/storage/drivers/csi.go

```diff
@@ -174,10 +174,10 @@ func (h *hostpathCSIDriver) PrepareTest(f *framework.Framework) (*testsuites.Per
 	node, err := e2enode.GetRandomReadySchedulableNode(cs)
 	framework.ExpectNoError(err)
 	config := &testsuites.PerTestConfig{
-		Driver:         h,
-		Prefix:         "hostpath",
-		Framework:      f,
-		ClientNodeName: node.Name,
+		Driver:             h,
+		Prefix:             "hostpath",
+		Framework:          f,
+		ClientNodeSelector: map[string]string{v1.LabelHostname: node.Name},
```

Comment:
Hostpath CSI driver supports topology now. I'll work on removing this in a separate PR

Reply:
Let's see if this works: https://github.com/kubernetes/kubernetes/pull/88050